extensions/wikia/Forum/css/Forum.scss

```diff
@@ -287,14 +287,16 @@
 	border-bottom: 0;
 	margin: 0 10px 0 0;
 	padding-bottom: 0;
-	// DAR-1382 overwrite right padding to make place for larger forum tally
-	padding-right: 260px;
+	padding-right: 0;
 	position: relative;
 
+	h1 {
+		margin: 0;
+		word-break: normal;
+	}
+
 	.header-right {
```

Comment:
.WikiaPage .WikiaPageHeader this looks like a generic class that touches every page
are we sure that Forum.css is not loaded on other pages? :)